.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### New Issue Checklist
```

Comment:
There's an HTML comment on top of the `PULL_REQUEST_TEMPLATE.md` down below, why not adding a similar one here too?

```
<!-- Thanks for taking the time to report your issue with _OHHTTPStubs_! When submitting your issue, please make sure to check the following boxes by putting an x in the appropriate [ ] so we can fully understand the context of your problem and help you better -->
```